front-end/src/App.js

```diff
@@ -1,24 +1,41 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
+import { Route } from 'react-router-dom';
+import { connect } from 'react-redux';
+
 import './styles/App.scss';
+import LandingPage from './components/LandingPage';
+import Callback from './components/Callback';
+import ScheduleView from './components/ScheduleView';
 
 class App extends Component {
   render() {
-    return (
-      <div className="App">
-        <h1>The Truth About Your Shape</h1>
-        <p>
-        A circle is a simple closed shape. It is the set of all points in a plane that are at a given distance from a given point, the centre; equivalently it is the curve traced out by a point that moves so that its distance from a given point is constant. The distance between any of the points and the centre is called the radius. This article is about circles in Euclidean geometry, and, in particular, the Euclidean plane, except where otherwise noted.
-        </p>
-
-        <form>
-          <input type="text">
-          </input>
-        </form>
-
-      </div>
-    );
+    if (this.props.auth.isAuthenticated()) {
```

Comment:
There are a couple different was to do this. But instead of writing out the same component twice. You can just check it on the line at the least.

``` {this.props.auth.isAuthenticated() ? <Route path="/schedule" component={ScheduleView} /> : null}```

Reply:
Must. resist. urge. to re-factor and make another commit on this branch lol. Right-on, I'll definitely want to pick your brain about writing good code.

Reply:
Lol just let me know